.github/workflows/pr-manual-merge.js

```diff
@@ -0,0 +1,79 @@
+const { slackNotification, getLocalConfigs } = require('./helpers.js');
+
+async function main({ github, context } = {}) {
+    if (!github || !context) {
+        throw new Error("GitHub context is missing. Ensure you are running in the correct environment.");
+    }
+
+    if (process.env.LOCAL_RUN) {
+        console.log("Local run detected. Loading local configurations...");
+        const localConfigs = getLocalConfigs();
+        github = localConfigs.github;
+        context = localConfigs.context;
```

Comment:
I'm not a fan of re-assigning parameters. Also if this was an object and u'd change something within an object, you'd change the original.
IMO Either use new variables or simply pass them into main.